winui/DataGrid/Row-Drag-and-Drop.md

```diff
@@ -0,0 +1,217 @@
+
+[WinUI DataGrid](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.SfDataGrid.html) allows drag and drop the rows by setting the `AllowDraggingRows` and [AllowDrop](https://docs.microsoft.com/en-us/dotnet/api/system.windows.uielement.allowdrop) property as `true`. You can also drag and drop the rows between datagrid and other controls such as `ListView` , `SfTreeView` and `SfTreeGrid`. 
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="sfDataGrid"
+                     AllowDraggingRows="True"
+                     AllowDrop="True"               
+                     AutoGenerateColumns="True"
+                     ItemsSource="{Binding Orders}" />
+{% endhighlight %}
+
+{% highlight c# %}
+this.sfDataGrid.AllowDraggingRows = true;
+this.sfDataGrid.AllowDrop = true;
+{% endhighlight %}
+{% endtabs %}
+
+While dropping, the dragged rows can be added above or below to the target record based on its drop position
+
+For example, if you drop record at the bottom of the target record, it will be added below the target record.
+
+<img src="Row-Drag-Drop-Images/RowDragDrop-Img1.png" alt="Drag and Drop Rows in WinUI DataGrid" width="100%" Height="Auto"/>
+
+
+If you drop above the target record, it will be added above the target record
+
+<img src="Row-Drag-Drop-Images/RowDragDrop-Img2.png" alt="Drag and Drop Rows in WinUI DataGrid" width="100%" Height="Auto"/>
+
+## Dragging multiple rows
+
+WinUI DataGrid (SfDataGrid) allows to select and drag multiple rows. To enable multiple selection, set the [SfDataGrid.SelectionMode](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Grids.SfGridBase.html#Syncfusion_UI_Xaml_Grids_SfGridBase_SelectionMode) as `Multiple` or `Extended`. 
+
+N> The drag selection cannot be performed while the `AllowDraggingRows` enabled as `true` in the SfDataGrid.
+
+<img src="Row-Drag-Drop-Images/RowDragDrop-Img3.png" alt="Dragging Multiple Rows in WinUI DataGrid" width="100%" Height="Auto"/>
+
+## Drag and drop events
+
+SfDataGrid triggers the following events when drag and drop:
+
+### Row Drag starting event
+
+`RowDragStarting` event occurs when you start to drag the records in the datagrid. The `DataGridRowDragStartingEventArgs` has the following member, which provides information for the `RowDragStarting` event.
+* `DraggingRows` : Gets the records which contains the data associated while dragging the rows.
+* `Cancel` : Gets or sets a value indicating whether the event is canceled or not. 
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.RowDragStarting += SfDataGrid_RowDragStarting;
+
+private void SfDataGrid_RowDragStarting(object sender, DataGridRowDragStartingEventArgs e)
+{
+    
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Row Drag over event
+
+`RowDragOver` event occurs continuously while record is dragged in the SfDataGrid. The `DataGridRowDragOverEventArgs` has the following members, which provide information for the `RowDragOver` event.
```

Comment:
`RowDragOver` event occurs continuously while dragging the record in the SfDataGrid.